### PR TITLE
Initial :new: :sparkles:

### PR DESCRIPTION
**Pull request by arnlaugsson**
Current version on https://www.coredata.is/setup - needs translation and styling. Styling and styling.

### CODE REVIEW

File: versions.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "CoreData Desktop": {
+	"version": "3.0.1",
+	"important": true,
+	"minVersion": "2.5.1",
+	"binary": "http://coredata.is/setup/coredata_desktop-${ARCH}-3.0.1.0.msi"
```
Comment (koddsson) [inline versions.json:6]: Should these be https?

File: versions.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    },
+
+    "CoreData Desktop .NET40": {
+        "version": "3.0.1",
+        "important": true,
```
Comment (koddsson) [inline versions.json:11]: What does this attribute do?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                cdd_version = val.version;
+            } else if (key.toUpperCase() === "COREDATA OUTLOOK") {
+                op_version = val.version;
+            } else if (key.toUpperCase() == "ZIMBRA ZIMLET") {
+                zz_version = val.version;
+            }
+        })
+    });
+
+    $(document).ready(function() {
```
Comment (koddsson) [inline index.html]: There should probably just be one `$(document).ready(...)` that will encapsulate all the stuff.

**Comment by koddsson** [comment]
General note: Can we replace all the "assets file" (bootstrap/font stuff) with their CDN counterpart instead of managing that asset ourselves?

File: versions.json (hunk 1 above)
Comment (arnlaugsson) [inline versions.json:6]: Yes, no doubt. This was originally just a demo file, copied from the server. I'll update the one on the server and this showcase version.

File: versions.json (hunk 2 above)
Comment (arnlaugsson) [inline versions.json:11]: CoreData Desktop updates can be marked as important, if they are then a notification will be shown to customers running CoreData Desktop versions that are older. Without 'important' flag the customer would have to run 'check for updates' to see the update.

File: index.html (hunk 3 above)
Comment (arnlaugsson) [inline index.html]: Makes sense.

**Comment by arnlaugsson** [comment]
@koddsson by all means. I originally copied a bunch of files into the repo, a cleanup is needed.

File: versions.json (hunk 1 above)
Comment (arnlaugsson) [inline versions.json:6]: (after making sure everything works post https).

**Comment by koddsson** [comment]
I'm gonna go head and make a PR to this PR :dog:

**Pull request merged**